src/locale/components/registrationForm/en.yml

```diff
@@ -0,0 +1,6 @@
+input_field_first_name: First name
+input_field_last_name: Last name
+input_field_email: E-mail
+input_field_phone: Phone number
+input_field_password: Password
+submit_button: Submit
```

Comment:
Instead of these flat lists of long field names, use the structure of YAML to group things (and camelCase):

```yaml
inputFields:
    firstName: First name
    lastName: Last name
    email: E-mail
    phone: Phone number
    password: Password
submitButton: Submit
```